client/src/components/customer/GoogleMap.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import GoogleMapReact from 'google-map-react'; // https://github.com/istarkov/google-map-react
+ // I placed the API key in a script tag within the index.html page
+const greatPlaceStyle = {
```

Comment:
Can we configure the style in the CSS file and just apply these as a class for this component?

For example: `className="restaurant-map"`
Then put the stuff in `greatPlaceStyle` inside `styles.css` as `.restaurant-map`

I think it'll be a bit cleaner this way.